rubin_sim/maf/metrics/snNSNMetric.py

```diff
@@ -81,17 +87,18 @@ def __init__(
         nexpCol="numExposures",
         vistimeCol="visitTime",
         seeingCol="seeingFwhmEff",
+        noteCol="note",
         season=[-1],
         coadd_night=True,
-        zmin=0.0,
-        zmax=1.2,
+        zmin=0.1,
+        zmax=0.5,
         zStep=0.03,
         daymaxStep=3.0,
         verbose=False,
         ploteffi=False,
         n_bef=3,
         n_aft=8,
-        snr_min=5.0,
+        snr_min=1.0,
         n_phase_min=1,
         n_phase_max=1,
```

Comment:
Can you make a comment about what snr_min means? Usually I would interpret this as saying that a minimum SNR of 1 is required for observations; that must be incorrect here though, so I am curious what this is intended to mean.

Reply:
snr_min is the minimum SNR of the light curve points used perform selections (n_min_phase, n_max_phase, number of epochs before and after max) and to estimate sigmaC.

Reply:
Hmm. Is this compatible with the fact that DM is only expecting to measure and release alerts on point sources with SNR=5 or higher? It could be, if forced photometry measurements of the source's location on the sky, which I think should be acquired after the source reaches snr=5, are useful.

Reply:
The metric assumes that the data are not coming from alerts but from image processing. We hope that SNR=1 will be possible.